example/mm_jobman.sh

```diff
@@ -140,7 +141,7 @@ create_download_commands() {
         cmd+="mkdir -p '$destination' && "
 
         # Add AWS download command
-        cmd+="aws s3 cp 's3://$source' '$destination' --recursive --quiet"
+        cmd+="/opt/aws/dist/aws s3 sync 's3://$source' '$destination'"
```

Comment:
1. No hard coded path (2nd time i raise this)
2. Can you clarify on the behavior of `sync`?  how does sync  behave in aws CLI? NOt sure if it can replace upload or download. Imagine we are in a collaborative environment where you and each of the other 10 collaborators upload folders from VM to the same root directory on S3. Then upload makes sense. If you sync  it, then it involves also downloading to your VM what others generated when you sync ? If that is the case then we need to be careful. Otherwise we shoul be fine (this is also the 2nd time I raise this, so please make sure it is addressed and not ignored)

Reply:
I'm open to adding another optin --sync if it actually means to sync on both ends